src/common/malloc.hpp

```diff
@@ -4,6 +4,8 @@
 #ifndef MALLOC_HPP
 #define MALLOC_HPP
 
+#include <config/core.hpp>
```

Comment:
just confirming, this for `custom/defines_pre.hpp` right?

Reply:
Yup! So that you can disable the memory manager there.